auth/presentation/src/test/java/com/sergius/auth/presentation/EmailPatternValidatorFake.kt

```diff
@@ -0,0 +1,16 @@
+package com.sergius.auth.presentation
+
+import com.sergius.domain.PatternValidator
+
+object EmailPatternValidatorFake: PatternValidator {
+
+    private var emailMatches = false
+
+    fun setEmailValidation(newValue: Boolean) {
+        emailMatches = newValue
+    }
```

Comment:
The typical Kotlin-like way would just be to make the `emailMatches` a public mutable property. Then you don't need the setter.